src/amo/reducers/collections.js

```diff
@@ -697,6 +703,32 @@ export const changeAddonCollectionsLoadingFlag = ({
   };
 };
 
+type LocalizeCollectionDetailParams = {|
+  detail: ExternalCollectionDetailWithLocalizedStrings,
+  lang: string,
+|};
+
+export const localizeCollectionDetail = (
+  { detail, lang }: LocalizeCollectionDetailParams
+): ExternalCollectionDetail => {
+  invariant(detail, 'detail is required for localizeCollectionDetail');
+  invariant(lang, 'lang is required for localizeCollectionDetail');
+
+  return {
```

Comment:
We should maybe add a comment to indicate that Flow forces us to repeat all fields, and that ideally we should not..